crit/explore.go

```diff
@@ -328,7 +328,7 @@ func (c *crit) ExploreRss() ([]*RssMap, error) {
 		for _, pagemapEntry := range pagemapImg.Entries[1:] {
 			pagemapData := pagemapEntry.Message.(*pagemap.PagemapEntry)
 			rss := Rss{
-				PhyAddr:  fmt.Sprintf("%x", pagemapData.GetVaddr()),
+				PhyAddr:  strconv.FormatUint(pagemapData.GetVaddr(), 16),
```

Comment:
@behouba Would you be able to add more information in the commit message to explain why these changes are needed? For example, you can add any errors or warnings reported by golangci-lint.

Reply:
Sure!